manifests/globals.pp

```diff
@@ -70,6 +71,13 @@
         /^5\./ => '8.1',
         default => undef,
       },
+      'RedHat' => '9.2',
+      default => $::operatingsystemrelease ? {
+        /^7\./ => '9.2',
+        /^6\./ => '9.2',
+        /^5\./ => '9.0',
+        default => undef,
+      },
```

Comment:
This block is a) duplication and b) wrong in that RHEL 5 vs 6 and 7 have different default postgresql versions and L74 would always set that to `'9.2'